src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -7,6 +7,7 @@
     "initialRows": 30,
     "launchMode": "default",
     "alwaysOnTop": false,
+    "inputServiceWarning":true,

```

Comment:
```suggestion
```

It is okay if we do not include this setting in the defaults files. Thank you for your attention to detail!

I will commit this and merge your PR -- no need to send another update.